C-Plus-Plus/math/Smallest divisible number.cpp

```diff
@@ -0,0 +1,51 @@
+/*Given a number n find the smallest number evenly divisible by each number 1 to n.
+
+##Example :
```

Comment:
Shift sample input/output at the end of the program. Also add space and time complexity of your approach.

Reply:
done